src/olympia/files/tasks.py

```diff
@@ -5,13 +5,15 @@
 
 from pathlib import Path
 
+from django.conf import settings
+from django.core.files.storage import default_storage as storage
```

Comment:
Why does this one use `default_storage` instead of `SafeStorage`? I notice this in a few files and I don't quite understand it.

Reply:
django provides a `default_storage` instance that uses `settings.DEFAULT_FILE_STORAGE` - that's the normal way (it's not initialized with any args so it ends up with a base_location of `settings.ROOT_STORAGE`).  If we want a non-standard storage location we can create our own instance.